### PR TITLE
Release v1.8.0 beta.1

### PR DESCRIPTION
**Pull request by kkaefer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,19 +1,46 @@
...
+* Fix line distances breaking gradient across tile boundaries ([#9220](https://github.com/mapbox/mapbox-gl-js/pull/9220))
+* Fix a bug where lines with duplicate endpoint could disappear at zoom 18+ ([#9218](https://github.com/mapbox/mapbox-gl-js/pull/9218))
+* Tile requests to Mapbox sources that don't use Mapbox URLs are now billed correctly ([#9217](https://github.com/mapbox/mapbox-gl-js/pull/9217))
+* Ctrl-click to drag rotate the map is disabled if the Alt, Cmd or Windows key is also pressed ([#9203](https://github.com/mapbox/mapbox-gl-js/pull/9203))
+* Pass errors to `getClusterExpansionZoom`, `getClusterChildren`, and `getClusterLeaves` callbacks ([#9251](https://github.com/mapbox/mapbox-gl-js/pull/9251))
+* Fix a rendering performance regression ([#9261](https://github.com/mapbox/mapbox-gl-js/pull/9261))
+* Fix visual artifact for `line-dasharray` ([#9246](https://github.com/mapbox/mapbox-gl-js/pull/9246))
+* Don't try to update the location indicator before the UI is set up ([#9288](https://github.com/mapbox/mapbox-gl-js/pull/9288)) (h/t [Meekohi](https://api.github.com/users/Meekohi))
```
Comment (asheemmamoowala) [inline CHANGELOG.md]: This fixes a bug introduced by a PR within this release, so can be skipped from the changelog

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,19 +1,46 @@
+## 1.8.0-beta.1
+
+### ✨ Features and improvements
+* Level-of-detail support for tile coverage ([#8975](https://github.com/mapbox/mapbox-gl-js/pull/8975))
```
Comment (asheemmamoowala) [inline CHANGELOG.md]: While the PR was merged, it states that:
> The maximum pitch angle is left unchanged so this PR has no immediate impact on behavior. coveringTiles should provide similiar results as the original implementation when pitch is < 60

Given then that the PR is an internal improvement only, does it warrant a changelog entry? If we think it does, might I suggest:

```suggestion
* Implemented a new tile coverage algorithm to enable level-of-detail support in a future release ([#8975](https://github.com/mapbox/mapbox-gl-js/pull/8975))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,19 +1,41 @@
...
+* Prevent empty buffers from being created for debug data when unused ([#9237](https://github.com/mapbox/mapbox-gl-js/pull/9237))
+* Add space between distance and unit in scale control ([#9276](https://github.com/mapbox/mapbox-gl-js/pull/9276)) (h/t [gely](https://api.github.com/users/gely)) and ([#9284](https://github.com/mapbox/mapbox-gl-js/pull/9284)) (h/t [pakastin](https://api.github.com/users/pakastin))
+* Add circle to GeolocateControl showing accuracy of position ([#9253](https://github.com/mapbox/mapbox-gl-js/pull/9253)) (h/t [Meekohi](https://api.github.com/users/Meekohi))
+* Implemented a new tile coverage algorithm to enable level-of-detail support in a future release ([#8975](https://github.com/mapbox/mapbox-gl-js/pull/8975))
+
+### 🐞 Bug fixes
+* `line-dasharray` is now ignored correctly when `line-pattern` is set ([#9189](https://github.com/mapbox/mapbox-gl-js/pull/9189))
+* Fix line distances breaking gradient across tile boundaries ([#9220](https://github.com/mapbox/mapbox-gl-js/pull/9220))
+* Fix a bug where lines with duplicate endpoint could disappear at zoom 18+ ([#9218](https://github.com/mapbox/mapbox-gl-js/pull/9218))
```
Comment (chloekraw) [inline CHANGELOG.md]: duplicate endpoints

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,19 +1,41 @@
...
+* Ctrl-click to drag rotate the map is disabled if the Alt, Cmd or Windows key is also pressed ([#9203](https://github.com/mapbox/mapbox-gl-js/pull/9203))
```
Comment (chloekraw) [inline CHANGELOG.md]: Add “fix” to the beginning of this entry

And maybe “being disabled” sounds better with the verb

Or:

“Enable ctrl-click to drag-rotate the map even if the alt, cmd or windows key is also pressed”

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,19 +1,41 @@
...
+* Pass errors to `getClusterExpansionZoom`, `getClusterChildren`, and `getClusterLeaves` callbacks ([#9251](https://github.com/mapbox/mapbox-gl-js/pull/9251))
+* Fix a rendering performance regression ([#9261](https://github.com/mapbox/mapbox-gl-js/pull/9261))
+* Fix visual artifact for `line-dasharray` ([#9246](https://github.com/mapbox/mapbox-gl-js/pull/9246))
+* Don't call `onSuccess` and `onError` callbacks after geolocate control was removed  ([#9291](https://github.com/mapbox/mapbox-gl-js/pull/9291))
```
Comment (andrewharvey) [inline CHANGELOG.md]: looks like my <changelog></changelog> entry in the PR didn't come through, what did I do wrong?

I suggest:

> Fixed a bug in the GeolocateControl which resulted in an error when `trackUserLocation` was `false` and the control was removed before the Geolocation API had returned a location

Comment (andrewharvey) [reply CHANGELOG.md]: In general I think the CHANGELOG should be written with the library user as the audience, as such it should focus on how those using the library may be affected by these changes, hence unless absolutely necessary it should avoid mentioning things that are internal to the library which don't matter to library users.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,19 +1,41 @@
+## 1.8.0-beta.1
+
+### ✨ Features and improvements
+* Reduce size of line atlas by removing unused channels ([#9232](https://github.com/mapbox/mapbox-gl-js/pull/9232))
+* Prevent empty buffers from being created for debug data when unused ([#9237](https://github.com/mapbox/mapbox-gl-js/pull/9237))
+* Add space between distance and unit in scale control ([#9276](https://github.com/mapbox/mapbox-gl-js/pull/9276)) (h/t [gely](https://api.github.com/users/gely)) and ([#9284](https://github.com/mapbox/mapbox-gl-js/pull/9284)) (h/t [pakastin](https://api.github.com/users/pakastin))
+* Add circle to GeolocateControl showing accuracy of position ([#9253](https://github.com/mapbox/mapbox-gl-js/pull/9253)) (h/t [Meekohi](https://api.github.com/users/Meekohi))
```
Comment (andrewharvey) [inline CHANGELOG.md]: Should we mention the actual option?

> Add a new option to GeolocateControl, `showAccuracyCircle` that shows the accuracy of the user's location as a transparent circle. Enabled by default.

**Pull request merged**